src/Interfaces.test.ts

```diff
@@ -8,14 +8,14 @@ const TheRaven: Work = {
   signature:
     '3045022100e020a7ffeffa5d40ffde618c6c861678e38de69fd377028ec57ad93893883b3702201f085284a9064bab7e1cd39349e65d136d8f67e4b6b897c3e7db6b400ed91034',
   type: ClaimType.Work,
-  dateCreated: new Date('2017-11-13T15:00:00.000Z'),
+  created: '2017-11-13T15:00:00.000Z',
```

Comment:
Can we clarify in our API/docs that we're expecting a string here, not a JS date object?

Reply:
None of the apis take this as input. We generate this. It IS documented here: https://github.com/poetapp/random/blob/master/Claims.md
Note that I haven't created a PR to replace `created` with `issuanceDate`